modules/client-configuration/pages/vhm-aws.adoc

```diff
@@ -40,3 +40,33 @@ Instances running on the Amazon public cloud report a UUID to the {productname}
 ----
 I1234567890abcdef0
 ----
+
+
+
+== AWS Permissions for Virtual Host Manager
+
+For security reasons, always grant the least privilege possible for a task to be performed.
+Thus using the Access Key for a user with excessive permissions to AWS is not advised.
```

Comment:
"Using an Access Key with excessive permissions for users connecting to AWS is not advised."

Reply:
```suggestion
Using an Access Key with excessive permissions for users connecting to AWS is not advised.
```